org.eclipse.lsp4j/.settings/org.eclipse.core.resources.prefs

```diff
@@ -1,7 +1,6 @@
 eclipse.preferences.version=1
 encoding//src/main/java=UTF-8
 encoding//src/main/resources=UTF-8
-encoding//src/main/xtend-gen=UTF-8
```

Comment:
Not quite sure about this change, it looks a bit incidental...

Anyway, it should not hurt, since `encoding/<project>=UTF-8` covers it all.

Just wondering what was the reason for this change.

Reply:
Thanks for noticing - I accidentally squashed that change into this PR. I have separated it out to a new PR now.